CRM/Case/XMLProcessor.php

```diff
@@ -83,7 +83,7 @@ public static function mungeCaseType($caseType) {
    *
    * @return array
    */
-  public function &allActivityTypes($indexName = TRUE, $all = FALSE) {
+  public static function &allActivityTypes($indexName = TRUE, $all = FALSE) {
```

Comment:
Might as well remove the php4-era leading ampersand while we're at it.

Reply:
I didn't check to see if anything that calls it changes the values, but doing a quick look now it doesn't seem like there's any, so yeah probably safe to remove.

Reply:
Given 52 test failures from removing the ampersand I'm going to put it back and can think about in future. It's probably not really 52 different things but can do in another PR.

Reply:
Weird. I didn't think that ampersand actually had any function in php7

Reply:
It has a purpose, but admittedly is probably not being used here. The php4 part was about performance before copy-on-write was a thing, mostly for the er... older people (cough cough...)

Thanks.